OpenEdXMobile/res/values/text_styles.xml

```diff
@@ -250,6 +250,10 @@
         <item name="android:ellipsize">end</item>
     </style>
 
+    <style name="profile_field_heading" parent="profile_field_description">
+        <item name="android:fontFamily">@font/roboto_mono_regular</item>

```

Comment:
Please create and use a reference for the font in `attrs.xml` i.e `secondaryFontRegular`.